lib/decanter/parser/json_parser.rb

```diff
@@ -0,0 +1,12 @@
+module Decanter
+  module Parser
+    class JsonParser < ValueParser
+
+      parser do |val, options|
+        raise Decanter::ParseError.new 'Expects a JSON string' if val.is_a?(Array) || val.is_a?(Hash)
```

Comment:
`JSON.parse` only accepts `string` type as an argument

Reply:
Forgive my ignorance here – does ActiveRecord require that the input to `create/update` is a hash? Or will it accept a serialized JSON string?

Reply:
What's the benefit of checking for an Array or a Hash as opposed to `is_a? String`?

Reply:
From some quick testing in the console, it looks like `JSON.parse` throws an error for anything that's not a string (including booleans, integers, etc).

Because of that, to me the most readable option seems like `unless is_a? String`, but I also wonder if a readable alternative could be rescuing from a JSON.parse error to tell the dev that the value is not valid JSON.